sdk/networkanalytics/azure-resourcemanager-networkanalytics/src/test/java/com/azure/resourcemanager/networkanalytics/NetworkAnalyticsManagerTests.java

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.networkanalytics;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.networkanalytics.models.DataProduct;
+import com.azure.resourcemanager.networkanalytics.models.DataProductProperties;
+import com.azure.resourcemanager.networkanalytics.models.ManagedServiceIdentity;
+import com.azure.resourcemanager.networkanalytics.models.ManagedServiceIdentityType;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class NetworkAnalyticsManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.create("centraluseuap", "Central US EUAP");
```

Comment:
why EUAP?

can it work in usually e.g. westus?

Reply:
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/aef2719d-d41d-4351-a4e1-2a25d3d0cc6c)
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/28dc5834-e770-47e1-a6dd-2dd27b551ab0)
The only locations that support `Microsoft.NetworkAnalytics/DataProducts` in the subscription I can use are `Central US EUAP` and `East US 2 EUAP`.

Reply:
What about other resources?

Reply:
https://learn.microsoft.com/en-us/azure/operator-insights/data-product-create?tabs=azure-cli

Reply:
None of it says EUAP?

But major problem is below the `@Disabled`
https://github.com/Azure/azure-sdk-for-java/pull/40402#discussion_r1620087502

If we disable the only test in live test, what we are running it for?